fe/fe-core/src/main/java/com/starrocks/scheduler/TaskBuilder.java

```diff
@@ -39,6 +39,7 @@ public static Task buildMvTask(MaterializedView materializedView, String dbName)
         task.setDbName(dbName);
         Map<String, String> taskProperties = Maps.newHashMap();
         taskProperties.put(MvTaskRunProcessor.MV_ID, String.valueOf(materializedView.getId()));
+        taskProperties.put(Constants.TASK_PROPERTY_DB_ID, String.valueOf(materializedView.getDbId()));
```

Comment:
why need dbId task has dbName properties, what about use this to find dbid?

Reply:
Ok, i will use dbName property in task.